distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -61,8 +63,10 @@ type Array
 
     == : Array -> Boolean
     == self that =
-        eq_at i = self.at i == that.at i
-        if self.length == that.length then 0.up_to self.length . all eq_at else Boolean.False
+        if Meta.is_same_object Array self then Meta.is_same_object Array that else
+            eq_at i = self.at i == that.at i
+            Common.Panic.catch_primitive handler=(_ -> Boolean.FALSE)
```

Comment:
Shouldn't `Boolean.FALSE` be just `False`?

Reply:
Obviously typo. fixed in [9ab4c64](https://github.com/enso-org/enso/pull/3667/commits/9ab4c644cbb091497aa6d80f7e1005fe783a8302). But I have to use `Boolean.False` otherwise:
```
Array.enso[68:56-68:60]: The name `False` could not be found.
Array.enso[69:89-69:93]: The name `False` could not be found.
```

Reply:
Normalized in [536cb03](https://github.com/enso-org/enso/pull/3667/commits/536cb0343892f982b9c0632f23ed407f2487679c)